src/main/java/com/shapesecurity/shift/es2017/parser/TokenType.java

```diff
@@ -107,6 +107,7 @@
     STRING(TokenClass.StringLiteral, "string"),
     REGEXP(TokenClass.RegularExpression, "regexp"),
     IDENTIFIER(TokenClass.Ident, "identifier"),
+    ESCAPED_KEYWORD(TokenClass.Ident, "escaped-keyword"),
```

Comment:
Feel a little weird about this being `Ident` instead of `Keyword`. (shift-parser-js has [`Keyword`](https://github.com/shapesecurity/shift-parser-js/blob/f973b067adbb7e72fe3a010596bc50c73a5e6f3c/src/tokenizer.js#L132).) Is there a reason for it? I don't think it affects much either way; I'd prefer to default to matching the JS implementation unless there's a reason not to. Or we could change it, if there's a strong argument for `Ident` being correct.